### ISSUE TITLE
Improve Docker image for tests and builds

### ISSUE THREAD

**Issue by marcinn** [MaSzyna-Reloaded/MaSzyna-API-wrapper#85]
There are two separate Docker images: 
* one for making builds of the libmaszyna library
* another one for running tests

Also the Dockerfile used for tests calls `scons` twice. First one just before copying `src` to store compiled `godot-cpp` in a docker image layer (for caching purposes). It works, but this trick is error prone and could stop working after some modifications of the SConstruct file. The SConstruct should be prepared to solve this case, or just `scons` should be called directly in `godot-cpp` directory.

As @JezSonic suggested in a discussion on Discord, a just one Docker image should be enough to do all tasks. 
I suggest avoiding the need to upload the Docker image to dockerhub.

**Comment by JezSonic** [comment]
Yeah, as long as the image will be cached like this one for tests, it should be enough for us :)
